src/Interpreters/DatabaseAndTableWithAlias.cpp

```diff
@@ -38,7 +38,7 @@ DatabaseAndTableWithAlias::DatabaseAndTableWithAlias(const ASTIdentifier & ident
     else if (identifier.name_parts.size() == 1)
         table = identifier.name_parts[0];
     else
-        throw Exception(ErrorCodes::INVALID_IDENTIFIER, "Invalid identifier");
+        throw Exception(ErrorCodes::INVALID_IDENTIFIER, "Invalid identifier \'{}\'", identifier.name());
```

Comment:
```suggestion
        throw Exception(ErrorCodes::INVALID_IDENTIFIER, "Invalid identifier {}", backQuote(identifier.name()));
```

What if the identified contains a single quote? (yes, nothing bad will happen, but the message will look ugly)